client-v2/src/components/FeedItem.tsx

```diff
@@ -112,7 +122,10 @@ const FeedItem = ({
         }}
       >
         {isLive && startCase(sectionName)}
-        {!isLive && (firstPublicationDate ? notLiveLabels.takendDown: notLiveLabels.draft)}
+        {!isLive &&
+          (firstPublicationDate
+            ? notLiveLabels.takendDown
+            : notLiveLabels.draft)}
```

Comment:
I wonder whether we can get ts-lint to hook into prettier? Quite a few of my PRs have these added in too. (Unless there was some meaning here and I've missed it!)

Reply:
awesome! 👍

Reply:
presuming the parens are the subject here, we do hook tslint into prettier (tslint-config-prettier) - in this case removing the parens would change the behaviour!

Reply:
also yes good re-use thanks to @vlbee!

Reply:
It's specifically the fact that when I run prettier on a file it'll make changes that `ts-lint` doesn't catch - will happily take a look given it seems to be me that's seen this the most 👍